src/pages/debrief/cat-c/components/vehicle-checks-card/__tests__/vehicle-checks-card.cat-c.spec.ts

```diff
@@ -20,7 +20,8 @@ import * as welshTranslations from '../../../../../../assets/i18n/cy.json';
 import * as englishTranslations from '../../../../../../assets/i18n/en.json';
 import { QuestionResult } from '@dvsa/mes-test-schema/categories/common';
 import { PopulateTestCategory } from '../../../../../../modules/tests/category/category.actions';
-import { PopulateCandidateDetails } from '../../../../../../modules/tests/journal-data/candidate/candidate.actions';
+import { PopulateCandidateDetails }
+  from '../../../../../../modules/tests/journal-data/cat-b/candidate/candidate.actions';
```

Comment:
cat-b ... should this be cat c?

Reply:
changed the category and the imports as well